courses-portlet-webapp/src/main/webapp/css/degree-progress.css

```diff
@@ -1,215 +0,0 @@
-.degree-progress .fl-flex-2 {
```

Comment:
You can add those CSS files back in. The changes made only affect MyCourses (for the time being). I can go through the other portlets and add Bootstrap styles quickly, if need be.